src/applications/edu-benefits/0993/containers/SinglePageFormApp.jsx

```diff
@@ -38,18 +34,6 @@ class FormApp extends React.Component {
       }
     }
 
-    // Show nav only if we're not on the intro, form-saved, error, or confirmation page
-    // Also add form classes only if on an actual form page
-    if (isInProgress(trimmedPathname)) {
-      formNav = <FormNav formData={formData} formConfig={formConfig} currentPath={trimmedPathname}/>;
-
-      renderedChildren = (
-        <div className="progress-box progress-box-schemaform">
```

Comment:
These classes were reincorporated in a [later commit](https://github.com/department-of-veterans-affairs/vets-website/pull/7989/commits/3a6e549a521640f951d9ab733bcc4b44e125bd37), as they ensure that the back button is full width, and that the form is indented so that validation styles are visible.